web/themes/custom/cloudy/templates/content/council-document/node--council-document--search-result.html.twig

```diff
@@ -57,16 +57,18 @@
         {% endif %}
       </div>
     </div>
+    {% if content.field_council_document_status.0 %}
     <div class="row g-0">
       <div class="col-12 col-md">
         <div class="field field--label-inline d-flex">
           <div class="field__label">Disposition</div>
           <div class="field__item">
-            {{ content.field_council_document_status.0 }} {{ content.field_amended_by_council[0] }}
+            {{ content.field_council_document_status.0 }} {{ content.field_amended_by_council.0 }}
```

Comment:
Just curious: what's the difference between field_amended_by_council[0] and field_amended_by_council.0?

Reply:
They both function the same: https://twig.symfony.com/doc/3.x/templates.html#other-operators

> `., []`: Gets an attribute of a variable.

But generally we use `.0` in our twig templates so while I was in that file I noticed it and edited it to be consistent.